settings.php

```diff
@@ -18,39 +18,64 @@ function page_optimize_settings_page() {
 }
 
 function page_optimize_settings_section() {
-	// TODO: Add description
+	_e( 'Concat JavaScript and CSS for faster site loading and lower number of requests. Scripts are grouped by the original placement.' );
```

Comment:
I am still getting familiar with WordPress. Why use `_e()` instead of `__()`?

Reply:
`_e()` == `echo __()`

Reply:
don't know how I missed that. thank you